src/components/MenuPanel/MenuPanel.js

```diff
@@ -171,55 +186,76 @@ class MenuPanel extends React.PureComponent {
                     : this.renderAppGroup(app, false)
                 )}
               </div>
-              <SystemAppsToggle onClick={this.handleToggleSystemApps}>
-                <h1
-                  style={{
-                    display: 'flex',
-                    justifyContent: 'space-between',
-                    alignItems: 'flex-end',
-                  }}
-                >
-                  <span>System</span>
-                  <span
-                    css={`
-                      transform: rotate(${systemAppsOpened ? 180 : 0}deg);
-                      position: relative;
-                      top: ${systemAppsOpened ? -5 : 0}px;
-                      font-size: 7px;
-                      opacity: 0.7;
-                    `}
-                  >
-                    <IconArrow />
-                  </span>
-                </h1>
-              </SystemAppsToggle>
-              <Transition
-                items={systemAppsOpened}
-                config={springs.swift}
-                from={{ height: 0 }}
-                enter={{ height: 'auto' }}
-                leave={{ height: 0 }}
+              <SystemAppsTransition
+                config={springs.smooth}
+                state={systemAppsOpened ? 'show' : 'hide'}
                 immediate={!animate}
                 onFrame={this.updateScrollVisible}
                 native
               >
-                {show =>
-                  show &&
-                  (props => (
-                    <animated.div style={{ ...props, overflow: 'hidden' }}>
-                      {systemApps.map(app => this.renderAppGroup(app, true))}
-                    </animated.div>
-                  ))
-                }
-              </Transition>
+                {({ openProgress, showBorder }) => (
+                  <div>
+                    <SystemAppsToggle onClick={this.handleToggleSystemApps}>
+                      <animated.div
+                        style={{
+                          position: 'absolute',
+                          height: '1px',
+                          left: '0',
+                          right: '0',
+                          bottom: '0',
+                          boxShadow: '0 1px 1px rgba(0, 0, 0, 0.1)',
+                          opacity: showBorder,
```

Comment:
I think this is what's causing the screenshot below, but I find this box shadow quite distracting on the open/close transition:

<img width="175" alt="Screen Shot 2019-04-09 at 8 15 02 PM" src="https://user-images.githubusercontent.com/4166642/55824511-44931f00-5b04-11e9-8565-980226140a8e.png">

Reply:
Yes, the idea is to elevate the toggle, so that the content slides from underneath it, like a drawer. I made it more subtle + in a single step (so that the transition reacts instantly): it should work better now :+1: